src/StakingNode.sol

```diff
@@ -113,14 +118,28 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
 
     /**
      * @dev Amount of shares queued for withdrawal (no longer active in staking). 1 share == 1 ETH.
-     * Increases when calling queueWithdrawals, and decreases when calling completeQueuedWithdrawals.
+     * Increases when calling queueWithdrawals, and decreases when calling completeQueuedWithdrawals and on slashing
      */
     uint256 public queuedSharesAmount;
 
+    /**
+     * @dev The address of the operator that this staking node is delegated to.
+     */
     address public delegatedTo;
 
     /**
-     * @dev Allows only a whitelisted address to configure the contract
+     * @dev The amount of shares queued for withdrawal that were queued before the upgrade
+     */
+    uint256 public legacyQueuedSharesAmount;
```

Comment:
can we call this something more specific as its specific to the slashing upgrade

preELIP002QueuedSharesAmount ?

or any other code name